### ISSUE TITLE
Add boundary for new unit KAWW, once available

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#783]
See #781.

Hopefully available in the next Lands release.

**Comment by chadlawlis** [comment]
From Lands:

> Currently the Northeast region is estimating 6 months before
> completion of the land status mapping for KAWW.

**Comment by chadlawlis** [comment]
Boundary has been made available in the 20170630 Lands release.

Updated in DB via:

```sql
UPDATE parks_poly SET geom_poly = (SELECT wkb_geometry FROM lands_20170930 WHERE unit_code = 'KAWW'), data_source = 'Land Resources Division', min_zoom_poly = 14, min_zoom_border = 14, min_zoom_tintband = 14 WHERE unit_id = 499;

UPDATE parks SET unit_area = (SELECT ST_Area(geom_poly) FROM parks_poly WHERE unit_id = 499), nps_official = 'true' WHERE unit_id = 499;

UPDATE parks_point SET geom_point = (SELECT ST_Centroid(geom_poly) FROM parks_poly WHERE unit_id = 499) WHERE unit_id = 499;

UPDATE parks_label SET geom_label = (SELECT geom_point FROM parks_point WHERE unit_id = 499), min_zoom_label = 14, min_zoom_label_long = 14, min_zoom_label_center = 14, max_zoom_label = 22, ldir = 'NE' WHERE unit_id = 499;
```

Will go live once boundaries are regenerated. Will likely require coordination with @jakecoolidge to ensure cartographic attributes are set properly.

**Comment by chadlawlis** [comment]
Updated a few cartographic attributes for better rendering and labeling:

```sql
UPDATE parks_poly SET min_zoom_poly = 7, min_zoom_border = 12, min_zoom_tintband = 13 WHERE unit_id = 499;
UPDATE parks_label SET min_zoom_label = 6, min_zoom_label_long = 12 WHERE unit_id = 499;
```

This is now live.
